app/src/main/java/org/openedx/app/AppRouter.kt

```diff
@@ -127,37 +127,50 @@ class AppRouter : AuthRouter, DiscoveryRouter, DashboardRouter, CourseRouter, Di
     override fun navigateToCourseSubsections(
         fm: FragmentManager,
         courseId: String,
-        blockId: String,
-        mode: CourseViewMode,
-        descendantId: String?
+        subSectionId: String,
+        unitId: String?,
+        componentId: String?,
+        mode: CourseViewMode
     ) {
         replaceFragmentWithBackStack(
             fm,
-            CourseSectionFragment.newInstance(courseId, blockId, mode, descendantId)
+            CourseSectionFragment.newInstance(courseId, subSectionId, mode, unitId, componentId)
```

Comment:
Can you please use named arguments here, because there are a lot of arguments with the same type here:
```
CourseSectionFragment.newInstance(
    courseId = courseId, 
    subSectionId = subSectionId, 
    mode = mode, 
    unitId = unitId, 
    componentId = componentId,
)
```